keras_cv/layers/preprocessing/cut_mix.py

```diff
@@ -48,23 +47,14 @@ def _sample_from_beta(alpha, beta, shape):
         sample_beta = tf.random.gamma(shape, 1.0, beta=beta)
         return sample_alpha / (sample_alpha + sample_beta)
 
-    def call(self, images, labels, training=True):
-        """call method for the CutMix layer.
-
-        Args:
-            images: Tensor representing images of shape:
-                [batch_size, width, height, channels], with dtype tf.float32.
-            labels: One hot encoded tensor of labels for the images, with dtype
-                tf.float32.
-        Returns:
-            images: augmented images, same shape as input.
-            labels: updated labels with both label smoothing and the cutmix updates
-                applied.
-        """
-        # pylint: disable=g-long-lambda
-        cutmix_augment = lambda: self._update_labels(*self._cutmix(images, labels))
-        no_augment = lambda: (images, labels)
-        return tf.cond(tf.cast(training, tf.bool), cutmix_augment, no_augment)
+    def _batch_augment(self, inputs):
+        images = inputs.get('images', None)
+        labels = inputs.get('labels', None)
+        if images is None or labels is None:
+            raise ValueError(
+                'CutMix is expecting both images and labels as inputs.'
```

Comment:
nit:  f"CutMix expects inputs in a dictionary with format {"images": images, "labels": labels}.

Got: inputs={inputs}"

This is more descriptive of what users should pass

Reply:
Done.